### ISSUE TITLE
Accessibility: Font size is fixed

### ISSUE THREAD

**Issue by elrayle** [cul-it/exhibits-library-cornell-edu#489]
### Description

Accessibility: Font size is fixed

The complaint is for the `<p>` tag which is fixed to `16px`.  It wants the `font-size` to be expressed as a `%` or using `em`.

A quick search of `app/assets/stylesheets` reveals a lot of instance of `font-size` being defined using `px`.

### Page with this error

https://exhibits-prod-1.library.cornell.edu/fashion-in-transit/feature/visualizing-transit

Example: First paragraph after Visualizing Transit title

### File defining the style

https://github.com/cul-it/exhibits-library-cornell-edu/blob/6e9745f91ccd81a8f0691f0e67dbe799733f7c43/app/assets/stylesheets/exhibits/_base.scss#L35-L38

### Solution

Need to replace most if not all instances of `font-size` being defined with `px` to be defined with `em` instead. 

### Impacts

At the writing of this issue, instances of this error occur in...
* 3,193 pages
* 16,928 occurrences

A fix of all occurrences will raise the score by 1.91 points.

**Comment by elrayle** [comment]
You may want to review the feedback in Issue #490.  There are some comments on font size.

**Comment by mhk33** [comment]
https://my2.siteimprove.com/Accessibility/1081272/NextGen/Issue?ruleId=74&pageSegments=&issueKind=1&conformance=&siteGoal=

**Comment by mhk33** [comment]
I manually checked a couple of the pages for this issue in SiteImprove and the changes I made cleared them. So hopefully this should be clear (or much better) on the next SI crawl.

**Comment by mhk33** [comment]
This is fixed in SiteImprove NextGen now.
